### DISCUSSION TITLE
在通过vite打包构建发生了问题。

### DISCUSSION THREAD

**Discussion by 752134655** [antvis/S2#2291] [Q&A]
提示了这一个问题Rollup failed to resolve import "ant-design-vue" from "node_modules/@antv/s2-vue/esm/index.js".，我想问s2-vue不是已经配置了ant-design-vue了么，我是需要在项目中引入ant-design-vue吗？是否有其他解决的方案？

**Comment by lijinke666** [comment]
`ant-design-vue` 是 `s2-vue` 的 `peerDependencies`, 而不是 `dependencies`, 所以需要你额外安装 

![image](https://github.com/antvis/S2/assets/21015895/7fa36630-8785-450b-a3fe-3ed5a788479d)

![image](https://github.com/antvis/S2/assets/21015895/bc06415e-37fb-4a74-8b7c-85855cc3fb4f)
